Source/Core/Core/TimePlayed.cpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <string>
+
+#include "Common/FileUtil.h"
+#include "Common/IniFile.h"
+#include "TimePlayed.h"
+
+// used for QT interface - general access to time played for games
+TimePlayed::TimePlayed()
+{
+  m_game_id = "None";
```

Comment:
I think if you want a default unset ID, an empty string makes more sense than 'None'.